bench/workgen/runner/cache_workload_update_trigger.py

```diff
@@ -0,0 +1,256 @@
+    print('----- End of Cache Eviction statistics -----', file=fh)
+    print(' ', file=fh)
+
+
+    stat_cursor.close()
+
+context = Context()
+# eviction_updates_trigger=30
+conn_config = ""
+conn_config += ",cache_size=10GB,eviction=(threads_max=8),log=(enabled=true),session_max=250,statistics=(fast),statistics_log=(wait=1,json=true),io_capacity=(total=30M)"   # explicitly added
+conn = context.wiredtiger_open("create," + conn_config)
+s = conn.open_session("")
+
+wtperf_table_config = "key_format=S,value_format=S," +\
+    "exclusive=true,allocation_size=4kb," +\
+    "internal_page_max=64kb,leaf_page_max=4kb,split_pct=100,"
+compress_table_config = ""
+table_config = "memory_page_max=10m,leaf_value_max=64MB,checksum=on,split_pct=90,type=file,log=(enabled=false),leaf_page_max=32k"
+tables = []
+table_count = 10
+for i in range(0, table_count):
+    tname = "table:test" + str(i)
+    table = Table(tname)
+    s.create(tname, wtperf_table_config +\
+             compress_table_config + table_config)
+    table.options.key_size = 10
+    table.options.value_size = 2000
+    tables.append(table)
+
+populate_threads = 4
+icount = 1000000
+# There are multiple tables to be filled during populate,
+# the icount is split between them all.
+pop_ops = Operation(Operation.OP_INSERT, tables[0])
+pop_ops = op_multi_table(pop_ops, tables)
+nops_per_thread = icount // (populate_threads * table_count)
+pop_thread = Thread(pop_ops * nops_per_thread)
+pop_workload = Workload(context, populate_threads * pop_thread)
+ret = pop_workload.run(conn)
+assert ret == 0, ret
+cache_eviction_file = context.args.home + "/cache_eviction.stat"
+get_cache_eviction_stats(s, cache_eviction_file)
+
+print('Populate complete')
+
+# Log like file, requires that logging be enabled in the connection config.
+log_name = "table:log"
+s.create(log_name, wtperf_table_config + "key_format=S,value_format=S," + compress_table_config + table_config + ",log=(enabled=true)")
+log_table = Table(log_name)
+
+ops = Operation(Operation.OP_UPDATE, tables[0])
+ops = txn(ops, 'isolation=snapshot')
+# use_commit_timestamp - Commit the transaction with commit_timestamp.
+#ops.transaction.use_commit_timestamp = True
+ops = op_multi_table(ops, tables, False)
+ops = op_log_like(ops, log_table, 0)
+thread_log_upd = Thread(ops)
+thread_log_upd.options.session_config="isolation=snapshot"
+# These operations include log_like operations, which will increase the number
+# of insert/update operations by a factor of 2.0. This may cause the
+# actual operations performed to be above the throttle.
+thread_log_upd.options.throttle=11
+thread_log_upd.options.throttle_burst=0
+
+ops = Operation(Operation.OP_SEARCH, tables[0])
+ops = txn(ops, 'isolation=snapshot')
+# use_commit_timestamp - Commit the transaction with commit_timestamp.
+#ops.transaction.use_commit_timestamp = True
+ops = op_multi_table(ops, tables, False)
+ops = op_log_like(ops, log_table, 0)
```

Comment:
Do we need a logging table thread in this Workload? I recommend removing unnecessary operations. I ran the Workload without it, and it still produced the expected output.

Reply:
Yes, I think it is more representative of server workloads, like oplog. Workload will meet the update trigger, but it is better if it is more representative of server/customer workloads. So I would like to keep it.